agent/bench-scripts/test-bin/fio

```diff
@@ -18,6 +18,7 @@ if [[ "${1%%=*}" == "--client" ]]; then
 	done
 elif [[ "${1}" == "--version" ]]; then 
 	echo "fio-${_BM_FIO_VER:-3.42}"
+	exit 0
 fi
 
 exit ${_BM_FIO_STS:-0}
```

Comment:
Are you sure you want to add the nested invocation of `exit`?  Without it, we'll still perform the invocation at line 24, and that one offers more interesting possibilities for testing.

Reply:
The new behavior introduced by removing the executable file check and executing the `fio -V` is that we check the return status of that command.  We now need to ensure `fio -V` is successful, while the non-version check is not.